reviews/api/views.py

```diff
@@ -16,6 +16,32 @@
 from .serializers import DecisionSerializer, ReviewSerializer
 
 
+def return_latest_decisions(objects):
+    # return latest decisions from each proposal
```

Comment:
I would say: return *only* the latest decision for each proposal in objects.

This could be a docstring rather than a comment.